oauth2_lib/settings.py

```diff
@@ -21,7 +21,7 @@ class Oauth2LibSettings(BaseSettings):
     ENVIRONMENT: str = "local"
     SERVICE_NAME: str = ""
     MUTATIONS_ENABLED: bool = False
-    ENVIRONMENT_IGNORE_MUTATION_DISABLED: list[str] = []
+    ENVIRONMENT_IGNORE_MUTATION_DISABLED: list[str] = ["development", "pre-prod"]
```

Comment:
I don't think we want `pre-prod` as a default of ignore mutation disabled

Reply:
👍 Adjusted